base/client.jl

```diff
@@ -117,7 +117,11 @@ function repl_cmd(cmd, out)
             if shell_name == ""
                 run(ignorestatus(cmd))
             elseif shell_name == "cmd"
-                run(ignorestatus(`$shell /c $(cmd)`))
+                run(ignorestatus(`$shell /c $cmd`))
+            elseif shell_name == "powershell"
+                run(ignorestatus(`$shell -Command $(shell_escape(cmd))`))
+            elseif shell_name == "busybox"
+                run(ignorestatus(`$shell $cmd`))
```

Comment:
> Do we need to do any special escaping for busybox on Windows?

Yes, for busybox now using full `sh` applet with `shell_escape()`

> If Julia bundles busybox on Windows, can we default to that (with the correct path if it is buried somewhere in the julia directory)?

Default is now set to busybox using full path to julia bin sub-directory.
(Full path might not be necessary since bin sub-directory is added to PATH)